src/librepgp/stream-armor.c

```diff
@@ -408,6 +419,7 @@ armored_src_close(pgp_source_t *src)
         return;
     }
 
+    (void) pgp_hash_finish(&param->crc_ctx, NULL);
```

Comment:
Imho it would be better to make it `if (param->crc_ctx->handle) { pgp_hash_finish(...) }`, or modify pgp_hash_finish to return if handle is NULL.

Reply:
I'm for the second option as caller of ``pgp_hash_finish`` shoudn't need to check "crc_ctx" state

Reply:
Agree, this would look much better.

Reply:
okis, done